bin/cli.js

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 const puppeteer = require('puppeteer');
-const devices = require('puppeteer/DeviceDescriptors');
+const devices = puppeteer.devices;
```

Comment:
Changed for braking change of puppeteer 5.0.0

see https://github.com/puppeteer/puppeteer/releases/tag/v5.0.0